src/main/java/seedu/address/model/entry/EntryContainsSearchTermsPredicate.java

```diff
@@ -0,0 +1,62 @@
+package seedu.address.model.entry;
+
+import java.util.function.Predicate;
+
+import seedu.address.commons.util.StringUtil;
+import seedu.address.logic.commands.FindCommand;
+import seedu.address.logic.commands.FindCommand.FindEntryDescriptor;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Tests that a {@code Entry}'s {@code Title} matches any of the keywords given.
+ */
+public class EntryContainsSearchTermsPredicate implements Predicate<Entry> {
+    private final FindCommand.FindEntryDescriptor findEntryDescriptor;
```

Comment:
Can `FindCommand.FindEntryDescriptor` be shortened to just `FindEntryDescriptor`?